src/parquet/column/statistics.h

```diff
@@ -202,23 +202,32 @@ void TypedRowGroupStatistics<ByteArrayType>::PlainEncode(const T& src, std::stri
 template <>
 void TypedRowGroupStatistics<ByteArrayType>::PlainDecode(const std::string& src, T* dst);
 
-using BoolStatistics = TypedRowGroupStatistics<BooleanType>;
-using Int32Statistics = TypedRowGroupStatistics<Int32Type>;
-using Int64Statistics = TypedRowGroupStatistics<Int64Type>;
-using Int96Statistics = TypedRowGroupStatistics<Int96Type>;
-using FloatStatistics = TypedRowGroupStatistics<FloatType>;
-using DoubleStatistics = TypedRowGroupStatistics<DoubleType>;
-using ByteArrayStatistics = TypedRowGroupStatistics<ByteArrayType>;
-using FLBAStatistics = TypedRowGroupStatistics<FLBAType>;
-
-extern template class TypedRowGroupStatistics<BooleanType>;
-extern template class TypedRowGroupStatistics<Int32Type>;
-extern template class TypedRowGroupStatistics<Int64Type>;
-extern template class TypedRowGroupStatistics<Int96Type>;
-extern template class TypedRowGroupStatistics<FloatType>;
-extern template class TypedRowGroupStatistics<DoubleType>;
-extern template class TypedRowGroupStatistics<ByteArrayType>;
-extern template class TypedRowGroupStatistics<FLBAType>;
+typedef TypedRowGroupStatistics<BooleanType> BoolStatistics;
+typedef TypedRowGroupStatistics<Int32Type> Int32Statistics;
+typedef TypedRowGroupStatistics<Int64Type> Int64Statistics;
+typedef TypedRowGroupStatistics<Int96Type> Int96Statistics;
+typedef TypedRowGroupStatistics<FloatType> FloatStatistics;
+typedef TypedRowGroupStatistics<DoubleType> DoubleStatistics;
+typedef TypedRowGroupStatistics<ByteArrayType> ByteArrayStatistics;
+typedef TypedRowGroupStatistics<FLBAType> FLBAStatistics;
```

Comment:
I guess we could add this to `clang-tidy` config: http://clang.llvm.org/extra/clang-tidy/checks/modernize-use-using.html